### PR TITLE
install curl in build_dep.sh

### PR DESCRIPTION
**Pull request by NekoKedama**
### What changes were proposed in this pull request?
To add curl in dep in order to make http_test work properly

### Why are the changes needed?
To install curl while users don't have it 

### Does this PR introduce any user-facing change?
NO(I THINK)

### How was this patch tested?
I uninstall the curl and do build_dep again,it shows the curl will be installed.Then I run http_test and it runs properly.

### CODE REVIEW

File: build_dep.sh [hunk 1]
```diff
@@ -35,7 +35,8 @@ function yum_install {
         readline-devel \
         maven \
         java-1.8.0-openjdk \
-        unzip
+        unzip \
+	curl
```
Comment (whitewum) [inline build_dep.sh]: make alignment

File: build_dep.sh [hunk 2]
```diff
@@ -71,7 +72,8 @@ function aptget_install {
         python \
         maven \
         openjdk-8-jdk \
-        unzip
+        unzip \
+	curl
```
Comment (whitewum) [inline build_dep.sh]: ditto

**Comment by laura-ding** [comment]
jenkins go

**Comment by dutor** [comment]
Jenkins go

**Pull request merged**